pkg/util/sys/device.go

```diff
@@ -146,6 +147,9 @@ func GetDevicePartitions(device string, executor exec.Executor) (partitions []Pa
 				p.Filesystem = v
 			}
 
+			partitions = append(partitions, p)
+		} else if strings.HasPrefix(name, cephLVPrefix) && props["TYPE"] == LVMType {
```

Comment:
Should this be checking if the LV does **not** have the `ceph--` prefix?

Reply:
Nop, if we detect a device that **has** the prefix then it's used and we add a fake partition so it gets skipped.

Reply:
ah got it